CSS/components/button.css

```diff
@@ -0,0 +1,57 @@
+.btn{
+    min-width: 5rem;
+    min-height: 1.5rem;
```

Comment:
you can use width max-content just a suggestion.

Reply:
Ya refactored the code and used max-content
Got to know about this new property thank you